src/client/pages/Landing/components/Card.tsx

```diff
@@ -128,11 +132,58 @@ const CardContainer: React.FC<{
   )
 }
 
-export const Card: React.FC<{
+export const CardHeadline = styled.h2<{ disabled?: boolean }>`
```

Comment:
Just for the sake of keeping the css before the react components, maybe put this above `CardContainer` :)